src/script/cpp_api/s_item.cpp

```diff
@@ -69,7 +69,12 @@ bool ScriptApiItem::item_OnPlace(ItemStack &item,
 
 	// Call function
 	LuaItemStack::create(L, item);
-	objectrefGetOrCreate(L, placer);
+
+	if (placer == nullptr)
```

Comment:
if (!placer)

Reply:
Done, but why?

Reply:
we prefer to use (!placer) to handle both (!NULL) and (!nullptr) which are equivalent, but nullptr != NULL and we don't have converted all code to nullptr

Reply:
Ok, thanks